docs/data/pages.ts

```diff
@@ -7,10 +7,10 @@ import dataGridComponentApi from './data-grid-component-api-pages';
 import pickersComponentApi from './date-pickers-component-api-pages';
 import chartsComponentApi from './charts-component-api-pages';
 
-const isPreview =
-  process.env.NODE_ENV === 'development' ||
-  process.env.CONTEXT === 'deploy-preview' ||
-  process.env.CONTEXT === 'branch-deploy';
+// const isPreview =
+//   process.env.NODE_ENV === 'development' ||
+//   process.env.CONTEXT === 'deploy-preview' ||
+//   process.env.CONTEXT === 'branch-deploy';
```

Comment:
`process.env.DEPLOY_ENV` already exists for this use case. Removed in 8757e19e01795eacd190a6027ef7bb2caccfa9bc.